src/ast/semantic_analyser.cpp

```diff
@@ -733,6 +733,14 @@ void SemanticAnalyser::visit(Call &call)
     }
     call.type = SizedType(Type::integer, 8);
   }
+  else if (call.func == "unwatch")
+  {
+    if (check_nargs(call, 1))
+      check_arg(call, Type::integer, 0);
+
```

Comment:
I think it should check  `send_signal` and `-c cmd` or `-p pid` here. Erroring here is nicer than aborting later